rust/stackablectl/src/cmds/demo.rs

```diff
@@ -155,18 +165,46 @@ impl DemoArgs {
 
         let transfer_client = xfer::Client::new_with(cache);
 
+        let release_files = cli.get_release_files().context(PathOrUrlParseSnafu)?;
+        let release_list = release::ReleaseList::build(&release_files, &transfer_client)
+            .await
+            .context(BuildListSnafu)?;
+
+        let release_branch = match &self.release {
```

Comment:
note: This block of could looks like it can be added as a function to the `ReleaseList` type. This way, we can save duplicate code.

Reply:
note: We might also want to confirm that the release which the user provided via `--release` actually exists.

Something similar to `release_list.get(release)` comes to mind: `ReleaseList::contains(release) -> bool` for example. If this function returns false, we could use the (currently unused) `NoSuchRelease` error variant.

Reply:
Done in 01915ed.